packages/arb-token-bridge-ui/src/util/RetryableUtils.ts

```diff
@@ -9,6 +11,18 @@ type GetRetryableTicketParams = {
   l2Signer: Signer
 }
 
+type GetRetryableTicketExpirationParams = {
+  l1TxHash: string
+  l1Provider: JsonRpcProvider
+  l2Provider: JsonRpcProvider
+}
+
+type RetryableTicketExpirationResponse = {
+  date: number
+  daysTillExpiry: number
+  isValid: boolean
```

Comment:
I think we can make these a bit more clear:

```suggestion
  expirationDate: number
  daysUntilExpired: number
  isExpired: boolean
```

Do we maybe want to differentiate between `isExpired` and `isLoading`?

Reply:
Yes, we can, and then technically there will be 3 more non-result flags - 
`isExpired`, `isLoading`, `isLoadingError` - should I go ahead with all three, or it will be an overkill?

Reply:
Done. I have added all 3 for most flexibility, let me know if it doesn't make sense and we can prune them out.